### DISCUSSION TITLE
How do I save a link to other repository in the left menu?

### DISCUSSION THREAD

**Discussion by walterbeech** [community/community#137592] [New to GitHub]
Hello! I am new to GitHub. Somehow I've managed to save a link here, under the "Repositories" heading, but I cannot find how I did it.

![image](https://github.com/user-attachments/assets/777f0a12-07d0-4b2f-a9e1-2e57891da7bc)

**Comment by GabrielAlmeidag** [comment]
@walterbeech 🎊🎊

To create a new repository on GitHub, you should click on the “+” icon in the upper-right corner of the page. Then, select “New repository” from the dropdown menu. Fill in the repository name and other necessary details. Finally, click on “Create repository” to save it.

If you want to save an existing repository, you can do so by starring it. Navigate to the repository you want to save and click the “Star” button located in the upper-right corner of the repository’s page.

To view all your repositories, including the ones you’ve starred, look at the menu on the left side of the GitHub page. Click on “Repositories” to see a list of all the repositories you have created or starred.

**Comment by walterbeech** [comment]
Hello, Gabriel, and thanks for the reply! I've tried starring the repository, but it didn't appear in the said menu.

**Comment by Sin-cy** [comment]
It might be ur browser cache. 
Try clearing it and see if it shows up. Because every other repo I created does show up there automatically without having to do anything. 
I should mention that I created the repo from all the available Create Repository Button and it shows up just normal. Also I have zero stars on all of my repos soooo... yeah. I think this is a totally different section from the stars one.
<img src="https://github.com/user-attachments/assets/5769ea8c-49db-4c5a-80f0-27bbebde8c80" width="300" alt="Capture-2567-09-12-160035">
